tests/dummy/app/templates/application.hbs

```diff
@@ -22,7 +22,13 @@
       <p>Since there is no way to automate screen readers for testing, this is an interactive script to help you run all of the tests inside of each screen reader.</p>
 
       <h2>Tests</h2>
-      <p>Each of these tests should be reviewed in every single OS/AT/Browser combination.</p>
+      <p>Each of these tests should be reviewed in the appropriate OS/AT/Browser combination. Common browser/AT combinations include:</p>
```

Comment:
I agree with you that we should direct people toward the realistic combinations. Wording it is a pain.

```diff
- <p>Each of these tests should be reviewed in the appropriate OS/AT/Browser combination. Common browser/AT combinations include:</p>
+ <p>Each of these tests should be reviewed in our recommended OS/AT/Browser combinations, specifically:</p>
```